Makefile

```diff
@@ -953,6 +956,15 @@ else
 	@echo Cannot run an astyle check, your system either does not have astyle, or it is too old.
 endif
 
+style-json: new_json_whitelist json_formatter
+	cat new_json_whitelist | xargs -L 1 ./json_formatter
```

Comment:
Useless use of `cat`. Can't you use the shells build-in mechanism (`xargs <file`) or `xargs` option `-a` for this?